app/models/invite_user_form.rb

```diff
@@ -3,9 +3,9 @@ class InviteUserForm
 
   attr_reader :email, :given_name, :family_name, :roles
 
-  validates_presence_of :email, :given_name, :family_name, :roles
+  validates_presence_of :email, :roles
 
-  validate :email_is_valid, :validate_roles
+  validate :email_is_valid, :validate_roles, :first_name_is_present, :last_name_is_present
```

Comment:
there's an easier way to customize the error message, you can still use the `validates_presence_of` method but you can pass in the message, e.g. `validates_presence_of :first_name, message: t('whatever.error_text')`.

Alternatively, we don't need to make any changes here and just use locales to overwrite the name of the field in locales like so:
```
en:
  activerecord:
    attributes:
      inviteuserform:
        given_name: "First name"
```

However, I think we could just rename everything to the new naming (first/last name) and only make sure on the API call to cognito it is given/family name...

Reply:
I wasn't sure if I should change it everywhere (apart from when we send to Cognito) or to just change it there on the view as `given_name` and `family_name` are used quite a lot throughout... will change now... and will localise the error text... should have done that before.

Reply:
@jakubmiarka I have amended the commit and changed the field names... `given_name` and `family_name` are still  being used by the `user` model and controller... I think that makes sense as they are fairly tied to cognito and also changing them and not cognito is proving difficult... though I can keep trying if you literally only want `given_name` to be sent and received from Cognito.

Reply:
yeah, looks good! ... Cognito naming should not be tied to/affect our model namings...

Reply:
@jakubmiarka I have updated it and changed the field naming in the user model. I think I have changed it as much as I can now. Let me know what you think :)